packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -100,6 +109,7 @@ export const Tooltip = ({
   ...otherProps
 }: TooltipProps) => {
   const [show, setShow] = useState(false);
+  const [delaySettings, setDelaySettings] = useState(delay);
```

Comment:
Why do we need to save this in a state?
Do we want to support delays being changed on the fly? Even if we want that, do we need to save it in a state

Reply:
You're right, we could've avoid using states from the beginning. Just refactored it without states.